gdcdictionary/schemas/virus_sequence_run_taxonomy.yaml

```diff
@@ -4,13 +4,13 @@ id: "virus_sequence_run_taxonomy"
 title: Virus Sequence Run Taxonomy
```

Comment:
the node names don't match the names you added to the PR description. i think a mapping (name of folder in original data => node name) would be useful

Reply:
I put the complete node title after the folder name in the updated PR description.

Reply:
could you do the same for sra-src, run, RAO 🙏

Reply:
sorry for forgetting those 3. It's updated now.